### ISSUE TITLE
Consider deleting classes Discretization, EllipticDiscretization

### ISSUE THREAD

**Issue by keileg** [pmgbergen/porepy#1039]
```[tasklist]
### Tasks
- [ ] numerics.interface_lows.elliptic_discretization.py
- [ ] numerics.discretization.py
```

**Comment by keileg** [comment]
Deleting EllipticDiscretization should be straightforward, and with #1035 implemented, we should be able to delete the full file. Note: several classes (mpfa, tpfa +++)  inherit from this class, and must be updated.

It is also tempting to delete the discretization class. From what I can see, this will deprive all discretizations of a `__repr__` method, which is not ideal, but it should have no further ramifications. I'm not sure what to do here.

**Comment by keileg** [comment]
Resolved by #1057
